core/src/main/resources/reference.conf

```diff
@@ -179,7 +179,14 @@ backend {
         String? docker_user
         """
         submit = "/bin/bash ${script}"
-        submit-docker = """docker run --rm ${"--user " + docker_user} -v ${cwd}:${docker_cwd} -i ${docker} /bin/bash ${docker_cwd}/execution/script"""
+        submit-docker = """
+        docker run \
+          --rm -i \
+          ${"--user " + docker_user} \
+          -v ${cwd}:${docker_cwd} \
+          ${docker} \
```

Comment:
ToL I kinda wish this variable name had the word "image" in it for clarity, but unless we support a new and deprecated name for this variable that would be a breaking change 🙁